autopush/router/webpush.py

```diff
@@ -72,7 +73,8 @@ def preflight_check(self, uaid, channel_id):
             self.ap_settings.message_tables[month_table].all_channels,
             uaid=uaid)
 
-        if not exists or channel_id not in chans:
+        if (not exists or channel_id.lower() not
+                in map(lambda x: normalize_id(x), chans)):
```

Comment:
We're prolly going to have to file a separate issue to replace the uaid lookup with a lookup that does a set membership test in dynamodb, so that we're not pulling the entire list of channels then checking them.

We can ensure the channels are normalized, by updating the migration code to normalize them on monthly rotation.